editor/actions.js

```diff
@@ -28,6 +28,22 @@ export function replaceBlocks( uids, blocks ) {
 	};
 }
 
+export function insertBlock( block, after ) {
+	return {
+		type: 'INSERT_BLOCK',
+		block,
+		after,
+	};
+}
+
+export function setBlockToInsert( slug, after ) {
```

Comment:
If we are only talking about the temporary blue line, I think we should call that `setInsertionPoint`